### ISSUE TITLE
Missing asset.party models when loading map from sbox.exe but shows up when using sbox-dev.

### ISSUE THREAD

**Issue by sugmagaming** [Facepunch/sbox-issues#3075]
### Describe the bug

When using the regular sbox exe and loading my map asset.party models do not show up. They work when using sbox-dev. Attached are screenshots of the difference. 

### To Reproduce

.

### Expected behavior

.

### Media/Files

![game](https://user-images.githubusercontent.com/15720147/228620107-f19b743c-bbd3-4e2a-91a2-b989cc28586f.png)
![editor](https://user-images.githubusercontent.com/15720147/228620133-b803310e-62b0-4d82-bc43-63825c7a7bf4.png)

**Comment by garrynewman** [comment]
Is this map local or compiled and on asset party?

**Comment by sugmagaming** [comment]
it's compiled and on asset party, but it is my map so i have it locally as well.

**Comment by garrynewman** [comment]
Are the errors when loading it locally or from asset party?

**Comment by sugmagaming** [comment]
from asset.party

**Comment by sugmagaming** [comment]
i just tested on sbox.exe and it doesn't show the assetparty models when loading my map locally or through asset.party, but on sbox-dev.exe it works loading the map from asset.party and locally.

**Comment by anthonysharpy** [comment]
getting the same issue

**Comment by garrynewman** [comment]
Yeah - cool - this is expected right now. There was previously a hack in place that made this work and I removed it.

**Comment by eagleonedevteam** [comment]
![image](https://user-images.githubusercontent.com/70182442/236662078-b8ce0096-9d02-47c1-8a42-c2f01ee2c114.png)

Still doing it sad panda :(

**Comment by anthonysharpy** [comment]
i noticed that in the sandbox game client, if you run a locally existing project it will have missing models/materials, but if you download and run the same project via asset.party (rename the local project folder to something else so that s&box no longer recognises it), it works
